meta_aggregation_api/providers/zerox_v1/zerox_provider.py

```diff
@@ -240,10 +240,11 @@ async def get_swap_quote(
 
     async def get_orders_by_trader(
         self,
-        taker_token: str,
-        maker_token: str,
+        *,
+        chain_id: str,
```

Comment:
Chain id type str?

Reply:
this got inferred by PyCharm from the `_api_path_builder(..., chain_id: str)` below

Reply:
fixed